object/init.go

```diff
@@ -36,11 +39,13 @@ func initBuiltInUser() {
 		Name:          "admin",
 		CreatedTime:   util.GetCurrentTime(),
 		Id:            util.GenerateId(),
+		Type:          "normal-user",
 		Password:      "123",
 		DisplayName:   "Admin",
 		Avatar:        "https://casbin.org/img/casbin.svg",
 		Email:         "admin@example.com",
 		Phone:         "1-12345678",
+		Address:       []string{},
```

Comment:
default chinese phone number, 11 digits and satisfies regular expressions, corresponds to `organization.PhonePrefix`=86

Reply:
fixed